src/processTargets/targets/BaseTarget.ts

```diff
@@ -87,7 +87,7 @@ export default abstract class BaseTarget implements Target {
     return "edit";
   }
 
-  getRemovalHighlightRange(): Range | undefined {
+  getRemovalHighlightRange(): Range {
```

Comment:
not sure why this was nullable.  we don't have any implementations that return null, so I made it required to remove noisy null checks